go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/ssvlabs/ssv
 
-go 1.22
+go 1.22.0
```

Comment:
Why do we need to specify patch version here?

Reply:
This is pretty much the result of me running `go mod tidy` - I have no idea why it added patch version, my guess would be it just makes it a bit more explicit what `go 1.22` actually means (it means the **min** version of golang needs to be 1.22.0)